arangodb-net-standard/CollectionApi/CollectionApiClient.cs

```diff
@@ -58,5 +58,30 @@ public async Task PutCollectionTruncateAsync(string collectionName)
             }
             await GetApiErrorException(response);
         }
+
+        public async Task<GetCollectionResponse> GetCollectionAsync(GetCollectionOptions options = null)
```

Comment:
`collectionName` is a path parameter, therefore it should just be exposed as `string collectionName` per our conventions.  You can remove `GetCollectionOptions` and replace with a single string argument.  It should not be optional.

Reply:
We have nothing to do with RAML but the first example here neatly illustrates the difference between a path param (or URI parameter) and a query param: https://dzone.com/articles/understanding-the-uri-param-and-query-param-with-r

Reply:
That's something I wanted to ask actually. Then we probably need to do the same for https://github.com/Actify-Inc/arangodb-net-standard/pull/22